client/flutter/android/fastlane/report.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm not sure this file needs to be in the repo.

Reply:
This is also a generated file. Will remove it